dragonfly/implementations/sensor_logger.py

```diff
@@ -49,10 +50,14 @@ def add_endpoint(self, endpoint):
 
 
     def this_consume(self, message, basic_deliver):
+        logger.debug("consuming message to: {}".format(basic_deliver.routing_key))
         ### Get the sensor name
+        if not basic_deliver.routing_key.split('.')[0] == self.prefix:
+            logger.warning("should not consume this message")
+            return
```

Comment:
I don't understand what this catches - if it's consuming isn't it on a bound queue and thus starting with the prefix?  This is more out of curiosity, because I know you've tested this.

Reply:
I think it is still binding `#` from somewhere that I didn't find (or maybe I changed this then fixed that and forgot to revert this)... going to take another look at this and something else that came up over the weekend.

Reply:
that was true when I added the lines, but should not be true now.